test/functionalTest/cases/1080_batch_query_context/batch_query_context.test

```diff
@@ -0,0 +1,1252 @@
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+V2 batch query
+
+--SHELL-INIT--
+dbInit CB
+brokerStart CB
+
+--SHELL--
+
+#
+# 01. Create an entity E of type T, with attributes A1=11,A4=14,A5=15
+# 02. Create an entity E of type T2, with attributes A2=22,A5=25
+# 03. Create an entity E3 of type T, with attributes A3=33,A4=34,A5=35
+#
+# 04. POST /v2/op/query, with empty payload
+# 05. POST /v2/op/query, with non-empty payload but without any of the three optional vector fields
+# 06. POST /v2/op/query, with 'entities' as an object, see error
+#
+# 07. POST /v2/op/query, with only E in payload, see 2 entities
+# 08. POST /v2/op/query, with only E3 in payload, see 1 entity
+# 09. POST /v2/op/query, with E and E3 in payload, see 3 entities
+# 10. POST /v2/op/query, with E.* in payload, see 3 entities
+# 11. POST /v2/op/query, with E3.* in payload, see 1 entity
+# 12. POST /v2/op/query, with F.* in payload, see no entities
+#
+# 13. POST /v2/op/query, with E.* and 'attributes' as an object, not an array - see error
+# 14. POST /v2/op/query, with E.* and 'attributes' as an array of integers, not strings - see error
+#
+# 15. POST /v2/op/query, with only A5 in payload, see 3 entities
+# 16. POST /v2/op/query, with E.* and A1 in payload, see 1 entity
+# 17. POST /v2/op/query, with E.* and A5 in payload, see 3 entities
+# 18. POST /v2/op/query, with E.* and A1,A2 in payload, see 2 entities
+#
+# 19. POST /v2/op/query, with 'scopes' as an object, not an array - see error
+# 20. POST /v2/op/query, with a scope being a vector, not an object - see error
+# 21. POST /v2/op/query, with a scope without 'type' - see error
+# 22. POST /v2/op/query, with a scope without 'value' - see error
+# 23. POST /v2/op/query, with a scope with an unknown field (not 'value' nor 'type') - see error
+# 24. POST /v2/op/query, with a scope whose 'type' is not a String - see error
+# 25. POST /v2/op/query, with a scope whose 'value' is not a String - see error
+#
+# 26. POST /v2/op/query, with E.* and in a scope q:A1==11, see E/T
+# 27. POST /v2/op/query, with E.* and in a scope q:type, see all 3 entities
+# 28. POST /v2/op/query, with E.* and in a scope q:!type, see NO entities
+# 29. Create an entity E5 without type, with attribute A5=45
+# 30. Create an entity E5 of type T4, with attribute A5=45
+# 31. POST /v2/op/query, with E.*, and with scopes q:type and q=A5==45, see E5/T4 only
+# 32. POST /v2/op/query, with E.* with scopes q:!type and q=A5==45, see E5/<notype> only
+#
+
```

Comment:
Split done in 56d642b.

About geo-scopes ... the PR has nothing to do with geo-scopes.
If you want, I can add a few tests anyway, just let me know ...